src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -81,7 +81,9 @@ protected ActionResult execute() {
         return createShowPageResult(Const.ViewURIs.ADMIN_SEARCH, data);
     }
     
-    private AdminSearchPageData putCourseNameIntoMap(List<StudentAttributes> students, List<InstructorAttributes> instructors, AdminSearchPageData data) {
+    private AdminSearchPageData putCourseNameIntoMap(List<StudentAttributes> students,
+                                                     List<InstructorAttributes> instructors,
+                                                     AdminSearchPageData data) {
```

Comment:
This is another way that we commonly use, if the method name is not that long (so long that it squished everything to the right).